app/styles/headers/_eighth.scss

```diff
@@ -126,7 +126,7 @@ $button_bg: #f78d78;
 
             .icon{
                 position: absolute;
-                left: -35px;
+                left: -45px;
```

Comment:
Don't understand why this's important.

Reply:
This adds more space between icon and text. It's not imortant but only for more beauty :)

Reply:
Than I agree.
Beaty is important!)